connectors/src/connectors/zendesk/temporal/workflows.ts

```diff
@@ -330,12 +330,13 @@ export async function zendeskCategorySyncWorkflow({
   let hasMore = true;
 
   while (hasMore) {
-    const result = await syncZendeskArticleBatchActivity({
+    const result = await syncZendeskArticlesBatchActivity({
       connectorId,
       categoryId,
       currentSyncDateMs,
       forceResync,
       cursor,
+      pageSize: ZENDESK_BATCH_SIZE,
```

Comment:
IMHO we should move the batch size into the activities to avoid having to bump the workflow version if we were to change it